components/resourceDetails/resourceSharing/addAgentButton/agentPickerModal/addWebIdButton/styles.js

```diff
@@ -24,6 +24,11 @@ import { createStyles } from "@solid/lit-prism-patterns";
 export default function styles(theme) {
   return createStyles(theme, ["icons", "button"], {
     button: {
+      [theme.breakpoints.down("xs")]: {
```

Comment:
I prefer to do mobile first for CSS, i.e. let the rules for smaller screen come first, and then add breakpoints for larger widths. Please flip `theme.breakpoints.down` into `theme.breakpoints.up`. (Not 100% necessary if it's hard, but would like to stay consistent.)